src/renderer/features/operation/init/ui/OperationHeader.tsx

```diff
@@ -46,6 +47,11 @@ export const OperationHeader = ({
   const multishardError = (isMultishard && errors.find((e) => e === OperationError.INVALID_FEE)) || undefined;
   const emptyError = errors.find((e) => e === OperationError.EMPTY_ERROR);
 
+  const availableShards =
+    walletUtils.isPolkadotVault(activeWallet) && accounts.length > 1
+      ? accounts.filter((a) => a.type !== AccountType.BASE)
```

Comment:
We have domain functions for every `account.type`, please use `accountUtils.isBaseAccount`